src/lwc/utilPlatformEventListener/utilPlatformEventListener.js

```diff
@@ -29,11 +47,55 @@ export default class PlatformEventListener extends LightningElement {
         setDebugFlag(this.isDebugFlagEnabled);
     }
 
+    classifyChannelName() {
+        let isFullName = this.channelName && this.channelName.includes('/event/');
+        let isNamespaceContext = this._namespacePrefix && this._namespacePrefix !== '';
+        let isChannelNamespaced = this.channelName.includes(this._namespacePrefix);
```

Comment:
I'm worried about false positives here if the string 'npsp' happens to occur in the event name ... I think however that if we include the "__" with the namespace that should be a safe check since that's usually a protected symbol?

Reply:
Also - are we worried about case-insensitive includes()?